### PR TITLE
[ApolloPagination] Update README

### PR DESCRIPTION
**Pull request by Iron-Ham**
Update the README to let folks know that this package may be subject to breaking changes.

### CODE REVIEW

**Review by AnthonyMDev — APPROVED** [review]

**Comment by calvincestari** [comment]
@Iron-Ham did you want to make anymore changes to this PR or can it be merged?

**Comment by Iron-Ham** [comment]
@calvincestari I think this is good to merge! When we get a bit closer to a proper release, I'll look into a more comprehensive README.

**Comment by calvincestari** [comment]
Looks like the gitleaks check is holding this up but I'm going to go ahead and merge it anyways.

**Pull request merged**
